javascript/ql/src/semmle/javascript/CanonicalNames.qll

```diff
@@ -307,7 +310,15 @@ class CanonicalFunctionName extends CanonicalName {
   /**
    * Gets a function with this canonical name.
    */
-  Function getADefinition() { ast_node_symbol(result, this) }
+  override Function getADefinition() { ast_node_symbol(result, this) }
+
+  /**
+   * Gets the callee expression in a function call or `new` expression that refers to a function
+   * with this canonical name.
+   */
+  override Expr getAnAccess() {
+    exists(InvokeExpr invk | ast_node_symbol(invk, this) | result = invk.getCallee())
```

Comment:
I'm wondering if we should include exprs that aren't `InvokeExpr`, to be able to find places where the function is passed as a callback:
```suggestion
    exists(InvokeExpr invk | ast_node_symbol(invk, this) | result = invk.getCallee())
    or
    ast_node_symbol(result, this) and
    not result instanceof InvokeExpr
```

Reply:
Oh, great idea!